README.md

```diff
@@ -116,4 +116,21 @@ clientConfig.setMaxRequests(60);
 
 profile.setHttpClientConfig(clientConfig);
 IAcsClient client = new DefaultAcsClient(profile);
+```
+
+## Timeout
+Request Settings -> Customer Settings -> Default, priority from high to low;
+The default ConnectTimeout is 5 seconds and the ReadTimeout is 10 seconds.
+```java
+//The Client timeout setting is valid for all current requests, with priority higher than the default

```

Comment:
The timeout setting of client is valid for all requests

Reply:
The Client Setting is valid for all requests, with priority higher than default and lower than request